frontend/ui/timeval.lua

```diff
@@ -150,9 +143,70 @@ Creates a new TimeVal object based on the current time.
 
 @treturn TimeVal
 ]]
-function TimeVal:now()
+function TimeVal:realtime()
     local sec, usec = util.gettime()
     return TimeVal:new{sec = sec, usec = usec}
```

Comment:
It's unrelated, on Linux, it's an ever-increasing clock (i.e., it will *never* go backwards, it may *only* report the exact same value multiple times), starting at 0 on the current cold boot, that stops ticking during suspend.

So, yeah, it's there's for feature-parity and documentation and in case some crazy person ever wants to try wall-to-monotic conversions ;).

Reply:
So, which should we use if one day, we wan't to store the data a bookmark or highlight is made as a timestamp (to be converted to a human YYYY-MM-DD date at display time)?

Reply:
I'd be inclined to suggest ISO 8601.

Reply:
The statistics plugin already uses wall clock time for this very reason (and, in fact, is pretty much the main bit of code left that does that explicitly ;)).

Reply:
(e.g., os.time(), util.gettime() or TimeVal:realtime()).

Reply:
> (e.g., os.time(), util.gettime() or TimeVal:realtime()).

I wouldn't mind a comment somewhere stating that all these 3 return the same kind of time, unlike all the others TimeVal methods that are to be used only for internal time/delay/interval management.